batchflow/pipeline.py

```diff
@@ -1510,6 +1510,7 @@ def _gen_batch(self, *args, **kwargs):
         prefetch = kwargs.pop('prefetch', 0)
         on_iter = kwargs.pop('on_iter', None)
         notifier = kwargs.pop('notifier', kwargs.pop('bar', None))
+        total = kwargs.pop('total', None)
```

Comment:
Let's issue a deprecated warning if `bar` is passed.

Reply:
Please, also fix notifier init - it creates DummyBar on else clause. The better behaviour would be to create DummyBar on None and issue a warning for unknown  value.